src/sentry/static/sentry/app/components/fileChange.jsx

```diff
@@ -26,8 +26,8 @@ const FileChange = React.createClass({
   },
 
   render() {
-    let {filename, authors, types} = this.props;
-    types = Array.from(types);
+    let {filename, authors} = this.props;
+    // types = Array.from(types);
```

Comment:
there's other commented out stuff in that file and i didn't want to step on any toes, it's recent release stuff